server/utils/EmbeddingEngines/openAi/index.js

```diff
@@ -30,7 +30,8 @@ class OpenAiEmbedder {
         new Promise((resolve) => {
           this.openai
             .createEmbedding({
-              model: "text-embedding-ada-002",
+              model:
+                process.env.EMBEDDING_MODEL_PREF || "text-embedding-ada-002",
```

Comment:
Lets do what LLMProviders do here and create a `this.model` property in the constructor and do the assignment during class creation.

Then we can just use `this.model` everywhere we need to and can access the model from the LLM via `llmProvider.embedder.model`

Reply:
Done